spec/comment_spec.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) do
+    User.create(
+      name: 'Loren',
+      photo: 'https://this-person-does-not-exist.com/en/download-page?image=genef4e9868ae582ca3061881b69d8fbeb1',
+      posts_counter: 0,
+      bio: 'I am Loren, and I love sport.'
+    )
+  end
+
+  let(:post) do
+    Post.create(
+      title: 'My first post',
+      author: user,
+      text: 'The motivation to become developer is very big.',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+
+  subject do
+    Comment.new(
+      text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+      post: post,
+      author: user
+    )
+  end
+
+  before { subject.save }
+
+  describe 'Functionality' do
+    context 'update comments_counter' do
+      it 'updates the comments_counter of the post' do
+        expect { subject.update_comments_counter }.to change { post.comments_counter }.from(0).to(1)
```

Comment:
[OPTIONAL] On my end 2 of your tests fails. Where you are testing the `comment_counter` and the `likes_counter` kindly look into to make sure that all your tests pass.

![image](https://user-images.githubusercontent.com/38088818/236528223-a68a8ff1-3355-4fac-8e89-b14427070d51.png)